client/components/Overview/Overview.jsx

```diff
@@ -6,28 +6,57 @@ import Sizes from './components/Sizes';
 import Quantity from './components/Quantity';
 import AddToCart from './components/AddToCart';
 import ImageGallery from './components/ImageGallery/ImageGallery';
+import ImageGalleryExpand from './components/ImageGalleryExpand/ImageGalleryExpand';
 
 const Overview = ({ currentProduct }) => {
   //  Current selection of items to be used in specific
   //  components that rely on others existing e.g. size depends on current style
-  const [currentStyle, setCurrentStyle] = useState({});
+  const [currentStyle, setCurrentStyle] = useState(null);
   const [currentSize, setCurrentSize] = useState(null);
   const [currentQuantity, setCurrentQuantity] = useState();
   const [addToCartNoSize, setAddToCartNoSize] = useState(false);
   const [value, setValue] = useState('DEFAULT');
-  const [imageArray, setImageArray] = useState([]);
+  // const [imageArray, setImageArray] = useState([]);
```

Comment:
Is imageArray now overviewModal in terms of content?

Reply:
Image array is being removed since it no longer is an accurate representation of each styles images. Initially it was the first photo in each style.photos array but I realized each style can have many photos (unique to that style). After refactoring it was deemed a disappointment and removed.